.github/workflows/deploy_hold.yml

```diff
@@ -0,0 +1,45 @@
+name: Deploy assets to AWS S3 with hold state setup in github's production environment
```

Comment:
for my edification how does the hold state work here?

Reply:
There's some explanation in the pr description, but basically you can specify an 'environment' field in a github workflow. This environment is created in repo's settings. I setup one call 'production' with a 'require reviewers' protection rule on for team slapshot as qualify members to approve before workflow start deployment!

Reply:
oh ok! I was a little confused but I think I kind of get it now